eslint.config.mjs

```diff
@@ -0,0 +1,131 @@
+import { fixupPluginRules } from "@eslint/compat";
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import importPlugin from "eslint-plugin-import";
+import globals from "globals";
+import tsParser from "@typescript-eslint/parser";
+import nextPlugin from "@next/eslint-plugin-next";
+import reactHooks from "eslint-plugin-react-hooks";
+
+export default [
+  {
+    ignores: [
+      "**/dist",
+      "**/build",
+      "**/node_modules",
+      "**/*.config.js",
+      "**/*.config.mjs",
+      "**/eslint.config.mjs",
+      "**/vite.config.ts",
+      "tailwind.config.js",
+      "postcss.config.js",
+      "**/*.js",
+      ".storybook/preview.ts",
+      ".storybook/main.ts",
```

Comment:
question: why not include these? I understand the build files like `dist`, but why files that we control?